src/bindgen/index.md

```diff
@@ -20,6 +20,8 @@ use Rust in a C/C++ codebase in the first place!
 
 ## Getting Set Up
 
+> **Note:** Bindgen **requires** clang. Installation instructions can be found [here](https://github.com/rust-lang-nursery/rust-bindgen/blob/master/book/src/requirements.md#installing-clang-39)
```

Comment:
Seems the code blocks aren't getting `hljs` added to the class
The code inside isn't getting the span class `hljs-keyword' and `hljs-string`
It's just a block of text inside.